src/components/organisms/AdminVenueView/components/Container/Container.tsx

```diff
@@ -0,0 +1,317 @@
+      let top = Math.round(item.top + delta.y);
+      if (snapToGrid) {
+        [left, top] = doSnapToGrid(left, top);
+      }
+
+      moveBox(item.id, left, top);
+    },
+  });
+
+  const renderSelectedRoom = (index: number) => {
+    return (
+      <>
+        <DraggableSubvenue
+          isResizable={resizable}
+          id={index.toString()}
+          imageStyle={iconImageStyle}
+          rounded={!!rounded}
+          {...boxes[index]}
+          onChangeSize={resizeBox(index.toString())}
+          lockAspectRatio={lockAspectRatio}
+        />
+        {imageDims && interactive && (
+          <CustomDragLayer
+            snapToGrid={!!snapToGrid}
+            rounded={!!rounded}
+            iconSize={boxes[Object.keys(boxes)[index]]}
+          />
+        )}
+      </>
+    );
+  };
+
+  return (
+    <div ref={drop} style={{ ...styles, ...containerStyle }}>
+      <ReactResizeDetector
+        handleWidth
+        handleHeight
+        onResize={(width, height) => setImageDims({ width, height })}
+      />
+      <img
+        alt="draggable background "
+        style={{
+          width: "100%",
+          ...backgroundImageStyle,
+        }}
+        src={backgroundImage}
+      />
+
+      {backgroundImage &&
+        rooms.map((room, index) =>
+          room === selectedRoom ? (
+            <div key={`${room.title}-${index}`}>
+              {renderSelectedRoom(index)}
+            </div>
+          ) : (
+            <div
+              className="map-preview__room"
+              key={`${room.title}-${index}`}
+              onClick={() => !selectedRoom && setSelectedRoom(room)}
+              style={{
+                position: "absolute",
+                top: `${room.y_percent}%`,
+                left: `${room.x_percent}%`,
+                width: `${room.width_percent}%`,
+                height: `${room.height_percent}%`,
+              }}
+            >
+              <img
+                style={{
+                  width: "100%",
+                  height: "100%",
+                  filter: room.isEnabled ? "none" : "grayscale(100%)",
+                  opacity: room.isEnabled ? 1 : 0.5,
+                  transition: "filter .3s ease",
+                }}
+                src={room.image_url}
+                alt="room banner"
+                title={room.title}
+              />
+              <div
+                style={{
+                  position: "absolute",
+                  left: "50%",
+                  transform: "translateX(-50%)",
+                  display: "flex",
+                  alignContent: "center",
+                  justifyContent: "center",
+                  backgroundColor: "rgba(0, 0, 0, 0.5)",
+                  borderRadius: "22px",
+                  padding: "6px 8px",
+                  fontSize: "0.8rem",
+                  transition: "all 400ms cubic-bezier(0.23, 1, 0.32, 1)",
+                  whiteSpace: "nowrap",
+                }}
+              >
```

Comment:
Why does it have inline styles?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/f8fa0c67c34ac626919737b4bbd112b4bd715553